modules/eks/variables.tf

```diff
@@ -205,3 +205,14 @@ variable "ssh_to_nodes_allowed_cidr_blocks" {
   type        = list(string)
   default     = null
 }
+
+variable "cluster_enabled_log_types" {
+  description = "List of log types that will be enabled for the EKS cluster. Can be a subset of ['api', 'audit', 'authenticator', 'controllerManager', 'scheduler'] or an empty list."
+  type        = list(string)
+  default     = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
+
+  validation {
+    condition     = length(var.cluster_enabled_log_types) == 0 || alltrue([for val in var.cluster_enabled_log_types : contains(["api", "audit", "authenticator", "controllerManager", "scheduler"], val)])
```

Comment:
```suggestion
        condition     = length(coalesce(var.cluster_enabled_log_types, [])) == 0 || alltrue([for val in coalesce(var.cluster_enabled_log_types, []) : contains(["api", "audit", "authenticator", "controllerManager", "scheduler"], val)])
```

Reply:
To support `null` as value

Reply:
According to the documentation, the default value for the array ([]) is an empty one. So, I am unsure whether we should also accept null values. As per the validation rules, we are only allowed to accept an empty array (as described in the providers' default) and the items that are allowed as per the documentation. @g-iannelli